### PR TITLE
Update cluster templating for openstack to use existing networks and subnets

### PR DESCRIPTION
**Pull request by erkanerol**
Towards https://github.com/giantswarm/roadmap/issues/773

Documents https://github.com/giantswarm/kubectl-gs/pull/684

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 1]
```diff
@@ -109,7 +109,9 @@ It supports the following flags:
 - `--dns-nameservers` (optional) - A list of DNS nameservers to be used to resolve external names.
 - `--external-network-id` - UUID of the external network to be used. Only required if multiple external networks are available.
 - `--failure-domain` - Failure domain of worker nodes.
-- `--node-cidr` - CIDR defining the IP range of cluster nodes.
+- `--node-cidr` - CIDR defining the IP range of cluster nodes. When used, new network and subnet will be created.
+- `--network-name` - Name of existing network for the cluster. Can be used when `--node-cidr` is empty.
```
Comment (pipo02mix) [inline src/content/ui-api/kubectl-gs/template-cluster.md:113]:
```suggestion
- `--network-name` (optional) - Name of existing network for the cluster. Can be used when `--node-cidr` is empty. 
```

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 2]
```diff
@@ -109,7 +109,9 @@ It supports the following flags:
 - `--dns-nameservers` (optional) - A list of DNS nameservers to be used to resolve external names.
 - `--external-network-id` - UUID of the external network to be used. Only required if multiple external networks are available.
 - `--failure-domain` - Failure domain of worker nodes.
-- `--node-cidr` - CIDR defining the IP range of cluster nodes.
+- `--node-cidr` - CIDR defining the IP range of cluster nodes. When used, new network and subnet will be created.
+- `--network-name` - Name of existing network for the cluster. Can be used when `--node-cidr` is empty. 
+- `--subnet-name` - Name of existing subnet for the cluster. Can be used when `--node-cidr` is empty.
```
Comment (pipo02mix) [inline src/content/ui-api/kubectl-gs/template-cluster.md:114]:
```suggestion
- `--subnet-name` (optional) - Name of existing subnet for the cluster. Can be used when `--node-cidr` is empty. 
```

**Review by pipo02mix — COMMENTED** [review]
Please update date in the page's metadata section
https://github.com/giantswarm/docs/pull/1334/files#diff-700f81e5a3872969f1071ea72a89164d65eff6bee3952b7ddca9e8ec7512af9cR15

File: src/content/ui-api/kubectl-gs/template-cluster.md (hunk 1 above)
Comment (erkanerol) [reply src/content/ui-api/kubectl-gs/template-cluster.md]: applied

File: src/content/ui-api/kubectl-gs/template-cluster.md (hunk 2 above)
Comment (erkanerol) [reply src/content/ui-api/kubectl-gs/template-cluster.md]: applied

**Comment by erkanerol** [comment]
@pipo02mix Thanks for the review. I fixed the issues you pointed.

**Review by pipo02mix — APPROVED** [review]

**Pull request merged**